packages/flame_3d/lib/src/camera/world_3d.dart

```diff
@@ -68,7 +73,8 @@ class World3D extends flame.World with flame.HasGameReference {
   }
 
   void _prepareDevice() {
-    device.lights = lights;
+    device.lightingInfo.ambient = config.ambientLight;
+    device.lightingInfo.lights = lights;
```

Comment:
I might make this a fixed-size array later

Reply:
We should have a todo or tracking issue for that :P

Reply:
it depends on how I am able to implement arrays in glsl, which I am still unclear about. but will add a todo